examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -41,11 +42,13 @@ CHIP_ERROR PairingCommand::Run()
     // Generate a random remote id so we don't end up reusing the same node id
     // for different nodes.
     NodeId randomId;
-    DRBG_get_bytes(reinterpret_cast<uint8_t *>(&randomId), sizeof(randomId));
-    ChipLogProgress(Controller, "Generated random node id: 0x" ChipLogFormatX64, ChipLogValueX64(randomId));
-    if (GetExecContext()->storage->SetRemoteNodeId(randomId) == CHIP_NO_ERROR)
+    if (Controller::DeviceCommissioner::GetRandomOperationalNodeId(&randomId) == CHIP_NO_ERROR)
```

Comment:
This should come from `src/controller/ExampleOperationalCredentialsIssuer.cpp` or such, and be recorded based on the allocated NodeID during commissioning. Is this something that already has a TODO?

Reply:
There are existing TODOs in terms of sorting out the API for commissioners, yes....

I could add an `ExampleOperationalCredentialsIssuer` bit here, if you  prefer.  That's exactly what we are doing here.

This _is_ the allocation of NodeID during commissioning, for this commissioner.

Reply:
Oh, I did not realize we already had ExampleOperationalCredentialsIssuer.  Yes, using that makes total sense.